tests/jsx-split-attrs/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -122,8 +122,7 @@ long_open_long_children = (
         colour="blue"
         size="large"
         submitLabel="Sign in with Google"
-      />
-      d
+      />d
```

Comment:
Yep, I agree! 

In my first prototype it did have the behaviour of ensuring that the you always had a line break before and after elements that were broken. I must have lost that behaviour in one of my many reworkings of the code.

I'll aim to fix this up, hopefully some time next week.